aten/src/ATen/native/LossMultiMargin.cpp

```diff
@@ -103,15 +103,15 @@ void multi_margin_loss_out_cpu_template(
     const Tensor& target,
     int p,
     const Scalar& margin,
-    const Tensor& weight,
+    const c10::optional<Tensor>& weight,
```

Comment:
`multi_margin_loss_shape_check` accepts an optional `Tensor` because `weight` is an optional on CUDA. So I change this (and elsewhere) to be consistent.